node/pkg/db/governor.go

```diff
@@ -152,19 +211,44 @@ func (d *Database) GetChainGovernorData(logger *zap.Logger) (transfers []*Transf
 			}
 
 			if IsPendingMsg(key) {
-				msg, err := common.UnmarshalMessagePublication(val)
+				p, err := UnmarshalPendingTransfer(val)
 				if err != nil {
 					return err
 				}
 
-				pending = append(pending, msg)
+				pending = append(pending, p)
 			} else if IsTransfer(key) {
 				v, err := UnmarshalTransfer(val)
 				if err != nil {
 					return err
 				}
 
 				transfers = append(transfers, v)
+			} else if isOldPendingMsg(key) {
+				msg, err := common.UnmarshalMessagePublication(val)
+				if err != nil {
+					return err
+				}
+
+				p := &PendingTransfer{ReleaseTime: msg.Timestamp.Add(time.Duration(time.Hour * 72)), Msg: *msg}
```

Comment:
I did not make this a constant here, because this is one off code that will be removed in the next release. However, I did create a constant for the duration that is used in the rest of the governor code.